libs/components/pages/src/lib/modules/needs-attention/needs-attention.component.ts

```diff
@@ -8,9 +15,29 @@ import { SkyActionHubNeedsAttention } from '../action-hub/types/action-hub-needs
   templateUrl: './needs-attention.component.html',
   styleUrls: ['./needs-attention.component.scss'],
 })
-export class SkyNeedsAttentionComponent {
+export class SkyNeedsAttentionComponent implements OnChanges {
   @Input()
   public items: SkyActionHubNeedsAttention[] | undefined;
 
   public readonly gutterSize: SkyFluidGridGutterSizeType = 'large';
+
+  #logService = inject(SkyLogService);
+  #haveLoggedDeprecationWarning = false;
+
+  public ngOnChanges(changes: SimpleChanges): void {
+    if (
+      !this.#haveLoggedDeprecationWarning &&
```

Comment:
I thiiiiink Paul updated the log service to include the logic to only log once in 8, and we don't currently have the logging turned on in 7